test/system/700-play.bats

```diff
@@ -196,6 +196,32 @@ EOF
     run_podman rm -a
 }
 
+@test "podman kube play read-only" {
+    YAML=$PODMAN_TMPDIR/test.yml
+    run_podman create --pod new:pod1 --name test1 $IMAGE touch /test
```

Comment:
Suggestion: `touch /testrw`. And on the next line, `touch /testro`. And on the next, `touch /tmp/testtmp`. In my debugging of the btrfs-not-tmpfs bug, I found that that simple change made reading the code much easier for me.